### DISCUSSION TITLE
the master branch is production ready?

### DISCUSSION THREAD

**Discussion by AllanClapClap** [rejemy/UltraLiteDB#6] [General]
I want to use UltraLiteDB in my Unity Game, the current master branch is production ready since the last release was in 2020?

**Comment by rejemy** [comment]
Yes, it’s in a good state. I haven’t been putting up “official” builds recently since I’ve been doing a lot of little quality of life changes and none of them felt big enough for a release.I’m currently using the current version in a big project and haven’t hit any issuesOn Sep 21, 2023, at 6:15 PM, AllanClapClap ***@***.***> wrote:﻿
I want to use UltraLiteDB in my Unity Game, the current master branch is production ready since the last release was in 2020?

**Reply by AllanClapClap** [reply]
Nice! tks.
